js/views/SurveyListPage.js

```diff
@@ -54,7 +54,7 @@ const SurveyListPage = React.createClass ({
   componentDidMount() {
     this.mountTimeStamp = Date.now()
 
-    // Update Survey List from Parse only once every 3 minutes
+    // Automatically update Survey List from Parse only once every 3 minutes
```

Comment:
Mhm. I've integrated the changes of this PR with the `master` updates in the latest branch. This was committed before those changes, and has been properly adjusted on the new one.